docs/crds/README.asciidoc

```diff
@@ -16,13 +16,15 @@ Package v1alpha2 contains API Schema definitions for the policies v1alpha2 API g
 .Resource Types
 - xref:{anchor_prefix}-github-com-kubewarden-kubewarden-controller-apis-policies-v1alpha2-clusteradmissionpolicy[$$ClusterAdmissionPolicy$$]
 - xref:{anchor_prefix}-github-com-kubewarden-kubewarden-controller-apis-policies-v1alpha2-clusteradmissionpolicylist[$$ClusterAdmissionPolicyList$$]
+- xref:{anchor_prefix}-github-com-kubewarden-kubewarden-controller-apis-policies-v1alpha2-policyserver[$$PolicyServer$$]
+- xref:{anchor_prefix}-github-com-kubewarden-kubewarden-controller-apis-policies-v1alpha2-policyserverlist[$$PolicyServerList$$]
 
 
 
 [id="{anchor_prefix}-github-com-kubewarden-kubewarden-controller-apis-policies-v1alpha2-clusteradmissionpolicy"]
 ==== ClusterAdmissionPolicy 
 
-ClusterAdmissionPolicy is the Schema for the clusteradmissionpolicies API
+nolint:lll ClusterAdmissionPolicy is the Schema for the clusteradmissionpolicies API
```

Comment:
The generator did take a nolint comment and wrote it as documentation though... 🤔

Reply:
Good catch, I fixed that by moving the `//nolint` comment right lower into the block